rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +18,22 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise exceptions.ValidationError(msg)
+                    raise exceptions.ValidationError(
+                        msg,
+                        code='authorization'
```

Comment:
Not overly keen but open to discussion. However let's keep that seperate to this - I'd suggest opening it as an incremental improvement if and when we get this agreed & merged.